exercises/practice/ledger/src/test/java/LedgerTest.java

```diff
@@ -227,13 +226,10 @@ public void givenInvalidLocale_ThenException() {
         var entries = new Ledger.LedgerEntry[0];
         var locale = "it-IT";
 
-        // when
-        IllegalArgumentException illegalArgumentException = assertThrows(
-            IllegalArgumentException.class,
-            () -> ledger.format(EUR_CURRENCY, locale, entries));
-
         // then
```

Comment:
You can remove the `// given` and `// then` comments since they don't make a lot of sense anymore with this style.